### DISCUSSION TITLE
Cant move the line plugin

### DISCUSSION THREAD

**Discussion by gpraj-works** [pdfme/pdfme#622] [Q&A]
Why can't move the line when designing PDF? Its working perfectly increase width height and can rotate but can't move line to another place. I clicked, dragged but no response. Am I using it wrong?

https://github.com/user-attachments/assets/4147aaea-fa7f-4a97-a0dd-f578a4e536c6

**Comment by hand-dot** [comment]
I can't repro...

https://github.com/user-attachments/assets/2030af94-2e39-4e07-a194-383f75c18779

**Reply by gpraj-works** [reply]
Thank you.
